waiter/specs/k8s-default-pod.edn

```diff
@@ -0,0 +1,41 @@
+{:kind "ReplicaSet"
+ :apiVersion "extensions/v1beta1"
+ :metadata {:name #waiter/param :app-name
+            :labels {:app #waiter/param :app-name
```

Comment:
I don't recommend changing the `:app` label because that seems to be a strong Kubernetes convention. "Service" means something specific in the Kubernetes ecosystem, so I've been trying to avoid that term when mapping into K8s.

Using `:k8s-name` rather than `:app-name` for the template-placeholder-variable here might make sense though (as you suggested below with the `service-id->app-name` function).

Reply:
Okay, `k8s-name` sounds good.

Reply:
Renamed.